Description/en-US.md

```diff
@@ -1,75 +1,74 @@
 
-[![Current review](https://img.youtube.com/vi/bb8G9X5Q_4I/maxresdefault.jpg)](https://www.youtube.com/watch?v=bb8G9X5Q_4I)
+[<img src="https://img.youtube.com/vi/bb8G9X5Q_4I/maxresdefault.jpg" alt="Current review" height="140"/> - **a YouTube showcase by Rexus**](https://www.youtube.com/watch?v=bb8G9X5Q_4I)
 
 ### Performance
 
-Fabulously Optimized is usually faster than OptiFine due to the inclusion of [modern performance mods][1] like Sodium and Lithium.
+Fabulously Optimized is usually faster than OptiFine thanks to [modern performance mods][1] such as Sodium, Lithium and more!
+
+### Compatibility
+
+Fabulously Optimized works on any world or server, [supports 5 launchers, is hosted on 3 sites][6] and [translated to many languages][7]. All of its mods don't change vanilla mechanics and are purely visual.
 
 ### OptiFine parity
 
-Fabulously Optimized [supports almost all OptiFine features][2], including shaders, connected textures and zoom. [Get a free cape!][3]
+Fabulously Optimized [supports almost all OptiFine features][2], including shaders, connected textures and zoom. [You can even get your own free cape!][3]
 
 ### Minor tweaks
 
-Fabulously Optimized [includes minor tweaks][4] that don't break the vanilla feel, such as increased chat history, windowed full screen mode and bugfixes to most common issues.
+Fabulously Optimized [includes a few minor tweaks][4] that don't break the vanilla feel, such as increased chat history, windowed fullscreen mode and fixes to most common bugs.
 
 ### Up to date
 
-Fabulously Optimized [receives fast updates][5] and is one of the first modpacks to upgrade to newer Minecraft versions.
+Fabulously Optimized [receives fast updates][5] and is, in fact, one of the first modpacks to upgrade to newer Minecraft versions.
 
-### Works everywhere
+### Built on Fabric
 
-Fabulously Optimized is designed to work on any world or server, [runs on 5 launchers, is hosted on 3 sites][6] and [supports many languages][7].
+Compared to Forge, Fabric is lightweight, updates and loads fast, and still has great mod support.
 
 ### Open development
 
-Fabulously Optimized [is open-source][8] and so are most of its mods. It is developed with ideas from the community, collaborating with mod developers to create a better user experience for everyone.
+Fabulously Optimized [is open-source][8] and so are most of its mods. We always welcome suggestions from the users and collaborate with mod developers to create a better user experience for everyone.
 
-### Built on Fabric
-
-Compared to Forge, Fabric updates and loads fast, does not include huge libraries and still has great mod support.
-
-### Helpful community
+### Community
 
 We have a growing community of friendly members who like to discuss the pack and help each other. [Join us!][10]
 
 ## FAQ
 
-> **Q**: How do I install? Update? Add mods? Something else?
-> 
-> **A**: [The answers are in our wiki.][11]
+- **Q**: How do I install/update/add mods/do something else?  
```

Comment:
Why not quote format? It looks good on Modrinth IMO, and CF site update is coming soon as well.

Reply:
Quotes gray out the text, that doesn't really look good

Reply:
Where does it gray out the text? All I see is black bars before each question-answer set:
![image](https://user-images.githubusercontent.com/8611110/200170998-41f4f65e-004d-4194-8b56-f57c638333fb.png)